spec/features/shelters/destroy_spec.rb

```diff
@@ -78,6 +79,44 @@
 
       expect(page).to_not have_css("#pet-#{pet_1.id}")
     end
+
+    it "I can delete a shelter and that will delete all its reviews" do
+
+      shelter_2 = Shelter.create!(
+        name: "delete me",
+        address: "do",
+        city: "it",
+        state: "you",
+        zip: "coward")
+      dog_city = Shelter.create!(
+        name: "Dog City",
+        address: "1923 Dog Ln",
+        city: "Doggington",
+        state: "CO",
+        zip: "80414")
+      img_review = Review.create!(
+        title: "omg!!!",
+        rating: 5,
+        content: "pets super amazin!!",
+        image: "https://cdn0.wideopenpets.com/wp-content/uploads/2016/04/valentines-8.jpg",
+        shelter: dog_city)
+      no_img_review = Review.create!(
+        title: "eh",
+        rating: 3,
+        content: "it's ok I guess",
+        shelter: dog_city)
+
+      dog_city.reviews << [img_review]
+      shelter_2.reviews << [no_img_review]
+
+      visit "/shelters"
+
```

Comment:
maybe add `expect(Review.first.title).to eq("omg!!!")` before deleting just to confirm it _was_ the first review